Update pkg_security.es.Rmd (#1)
elioestudiante/dev_guide: pkg_security.es.Rmd

## Miscellaneous

> Recomendamos the article [Ten quick tips for staying safe online](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008563) by Danielle Smalls and Greg Wilson.

Reviewer: ```suggestion
Recomendamos el artículo [Ten quick tips for staying safe online](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008563) by Danielle Smalls and Greg Wilson.
```

Reviewer 2: ```suggestion
Recomendamos el artículo [Ten quick tips for staying safe online](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008563) por Danielle Smalls and Greg Wilson.
```
